src/components/SimplePriceChart/SimplePriceChart.tsx

```diff
@@ -127,16 +132,23 @@ const MarketDataChart: React.FC<SimplePriceChartProps> = ({
             dot={false}
             stroke={'url(#gradient)'}
             strokeWidth={2}
+            animationEasing="ease"
+            animationDuration={800}
           />
           <YAxis
             stroke={theme.colors.grey[500]}
-            tickFormatter={(n) => '$' + formatFloats(n)}
+            tickFormatter={tickFormatter}
             axisLine={false}
             tickLine={false}
             mirror={true}
-            tick={{ fontFamily: 'Roboto Mono' }}
-            ticks={[minimumYAxisLabel, maxY + 5]}
-            domain={[minY - 10, maxY + 10]}
+            tick={{ fontFamily: '' }}
```

Comment:
I'm going to say either turn this back to Roboto Mono, or remove the `tick=` altogether. No point in keeping if its empty

Reply:
Fair point. Removing the whole line seems like the best approach.